src/blackd/__init__.py

```diff
@@ -11,13 +11,11 @@
     from aiohttp import web
     import aiohttp_cors
 except ImportError as ie:
-    print(
+    raise RuntimeError(
```

Comment:
Should this maybe continue to be an `ImportError` instead of changing it to a `RuntimeError`?